docs/setting-up/server/helm.md

```diff
@@ -13,32 +10,84 @@
 !!! note alert alert-primary ""
     Helm v3 is needed to run the following steps.
 
+Refer to [Kubernetes Supported versions](https://kubernetes.io/releases/version-skew-policy/#supported-versions) and [Helm Version Support Policy](https://helm.sh/docs/topics/version_skew/) to find the supported versions.
+
+PMM should be platform-agnostic, but it requires escalated privileges inside a container. It is necessary to have a `root` user inside the PMM container. Thus, PMM would not work for Kubernetes Platforms such as OpenShift or others that have hardened Security Context Constraints, for example:
```

Comment:
This can be run on openshift as well but with required SCC.

Reply:
yes, I think that what I wrote there ? that SCC is limiting it.